app.js

```diff
@@ -114,6 +132,34 @@ app.post("/:rideID?", function(req, resp){
 				});
 })
 
+
+
+app.post("/rideInfo/:rideId?", function(req, resp){
```

Comment:
Post will be made without the `:rideId`. So, just `"/rideInfo"` might be enough.